ts/IterableInboxEmptyState.tsx

```diff
@@ -3,17 +3,22 @@
 import React from 'react'
 import { View, Text, StyleSheet } from 'react-native'
 
-const IterableInboxEmptyState = () => {
-   const title = "No saved messages"
-   const subtitle = "Check again later!"
+type emptyStateProps = {
+   title: string,
+   body: string
+}
+
+const IterableInboxEmptyState = ({title, body} : emptyStateProps) => {
+   const defaultTitle = "No saved messages"
+   const defaultBody = "Check again later!" 
 
    return(
       <View style={styles.container}>
          <Text style={styles.title}>
-            {title}
+            {title ? title : defaultTitle}
          </Text>
          <Text style={styles.subtitle}>
```

Comment:
Let's rename this from `subtitle` to `body` in this file as well. The more consistent terms used, the less possible confusion in the future when we'll definitely forget about this code.